api/helpers/db_helper.py

```diff
@@ -1,14 +1,16 @@
 from api import db
 import traceback
+from utils.loggingutils import logger
 
 
 def save(data):
     try:
         db.session.add(data)
         db.session.commit()
+        logger.info("Data saved successfully")
     except Exception as e:
-        print("Error: " + str(e))
-        print(traceback.format_exc())
+        logger.error("Error: " + str(e))
+        logger.debug(traceback.format_exc())
```

Comment:
What this line do?

Reply:
In line 12 we are printing the error exception (e) in string format.
In line 13 we are getting the formatted traceback information of the current exception for debugging the error.